tool/e2e-tests/dispense/dispensePrescription.ts

```diff
@@ -14,4 +23,52 @@ describe("firefox", () => {
     await dispensePrescriptionUserJourney(driver)
     await checkMyPrescriptions(driver, "Dispensed Prescriptions", prescriptionId)
   })
+
```

Comment:
The functionality of these tests is covered by **claimPrescriptions.ts**, this also goes for new tests written in **releasePrescriptions.ts**.